### DISCUSSION TITLE
Could we "share" blog post URLs with you?

### DISCUSSION THREAD

**Discussion by ZaLiTHkA** [dailydotdev/daily#434] [Feature Requests]
I discovered a blog post on `dev.to` from back in January 2021, which _now today_ helped me understand something I've been battling with.. however, the post itself has gathered a mere 9 "likes" in that time.

this made me wonder if it would be possible, perhaps through an input on the Chrome Extension "new tab" view, to submit a blog post URL to be considered for inclusion in the feed..? then truly helpful posts, like the one I found here, could reach more people. (:

---

to be fair, this _would_ open up a channel for shameless self-promotion with people punting their own super-little-known blogs with potentially dodgy scripts..... but there are also domain "trust" ratings, and public APIs for checking them, and many other measures, so at least most harmful things could be avoided/caught/filtered out.

so what do you think? is this a viable idea..? :)

**Comment by nimrodkra** [comment]
Hi @ZaLiTHkA that's definitely a viable idea. In fact, that's one of the next things we're going to build according to our roadmap. Keep in mind, that given the "tricky" nature of such a feature (like the scenarios you've mentioned), it will likely start as a closed beta until we feel comfortable opening it up widely.

**Reply by ZaLiTHkA** [reply]
awesome.. that is brilliant news. :) the initial closed beta idea also makes perfect sense.

if this is already being tracked somewhere else, please do feel free to close this issue then.. 🖖🏼

**Reply by idoshamun** [reply]
Hi @ZaLiTHkA,

You probably already know that this feature is out and it's called Community Picks :)
https://docs.daily.dev/docs/key-features/community-picks

**Reply by ZaLiTHkA** [reply]
indeed I do, thanks @idoshamun.. I've already shared a few articles through the "Community Picks" feature. :) must admit, I had forgotten about _this_ particular GH discussion though.

any idea how to mark this as "resolved"? I don't see anything in the UI on my screen.. :/

**Reply by idoshamun** [reply]
There's no resolved button but I can lock the convo. Thanks a lot!
